urbanaccess/gtfs/network.py

```diff
@@ -124,55 +141,89 @@ def create_transit_net(gtfsfeeds_dfs, day,
         gtfsfeeds_dfs.stop_times_int = _time_difference(
             stop_times_df=gtfsfeeds_dfs.stop_times_int)
 
+    # direction_id is conditional, though
+    if 'direction_id' not in gtfsfeeds_df.trips.columns:
+        columns.remove('direction_id')
+    
+    trips_subgdf = gtfsfeeds_df.trips[columns]
+    calendar_selected_trips_df = tripschedualselector(
```

Comment:
`tripschedualselector` should be `_trip_schedule_selector` as defined here: https://github.com/UDST/urbanaccess/blob/2f37f8ea8337fd65d654e9506b3c76f753af22db/urbanaccess/gtfs/network.py#L180